v1/summary_transform_filter.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+const cType = require('content-type');
+const semver = require('semver');
+
+const TAGS_MATCH = /<\/?[a-zA-Z][\w-]*(?:\s+[a-zA-Z_-]+(?:=(?:"[^"]*"|'[^']*'))?)*\s*\/?>/g;
+const LATEST_1_VERSION = '1.1.2';
+
+function splitProfile(profile) {
+    const match = /^(.*)\/([0-9.]+)$/.exec(profile);
```

Comment:
Why `.*` ? `.+` seems more prudent since /` is required by the RegExp before the version